app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/ytdliability/taxable_gain_or_loss.scala.html

```diff
@@ -35,7 +35,7 @@
         returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link_govuk,
         caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk,
         govukRadios: GovukRadios,
-        govukInput: GovukInput
+        govukInput: GovukInput,
```

Comment:
would this trailing comma break compilation?

Reply:
🤦 I left it in accidentally. No, it doesn't break compilation. In general, I like to leave trailing commas in since they make the diff smaller (you're changing only your line and not surrounding ones). It was incidental and I'll have to edit this PR again it looks like, so will remove it.